simpleLooping/createListPowerOfBase.py

```diff
@@ -11,3 +11,21 @@
 # Ex:
 #   createListPowerOfBase(2, 4) # returns [1, 2, 4, 8]
 #   createListPowerOfBase(3, 3) # returns [1, 3, 9]
+from typing import List
+ 
+def createListPowerOfBase(base: int, powerCap: int) -> List[int]:
+    if powerCap == 0:
+        return []
+
+    if powerCap == 1:
+        return [base]
+
+    if base == 0:
```

Comment:
This will make the code work in correctly, when base = 0 and powerCap = 5, I would like to return [0, 0, 0, 0 ,0]. In fact, you don't need this if at all.